doc/source/reference/arrays.datetime.rst

```diff
@@ -119,6 +119,9 @@ simple datetime calculations.
     >>> np.timedelta64(1,'W') / np.timedelta64(1,'D')
     7.0
 
+    >>> np.timedelta64(1, 'us') % np.timedelta64(727, 'ns')
```

Comment:
I would have thought something like `np.timedelta64(1,'W') % np.timedelta64(10,'D')` would be a slightly clearer example, but not really important